server/eclipse-project/src/main/java/us/freeandfair/corla/model/CVRAuditInfo.java

```diff
@@ -243,7 +243,7 @@ public void setDiscrepancy(final Set<AuditReason> the_reasons) {
    * as a disagreement in a contest audited for that reason.
    */
   public Set<AuditReason> disagreement() {
-    return Collections.unmodifiableSet(my_discrepancy);
+    return Collections.unmodifiableSet(my_disagreement);
```

Comment:
Whoops.  That's a cut-and-paste error originally.  We would not have noticed since we were focused on discrepancies.